configs/config.protectli_cml_vp4630_vp4650

```diff
@@ -32,16 +32,17 @@ CONFIG_DEFAULT_CONSOLE_LOGLEVEL_0=y
 CONFIG_POST_IO_PORT=0x80
 CONFIG_PAYLOAD_EDK2=y
 CONFIG_EDK2_REPOSITORY="https://github.com/Dasharo/edk2"
-CONFIG_EDK2_TAG_OR_REV="e31b7a712735f0623f67149efca0103be6134234"
+CONFIG_EDK2_TAG_OR_REV="bd421b40a9fcdb139df77e2ab583bf7de2ceae74"
```

Comment:
VP2420 used c959ce8d30fc98b4d2dd56688580ba5e4a3290bd in the respective PR, why this is different here?